### PR TITLE
Adding a migration test file for the user data model

### LINKED ISSUE #28: Create the migration file for the user test data

### PR DESCRIPTION
**Pull request by LeonRabinovich**
The test file contains 3 users which will be added to our DB after running `pipenv run python manage.py migrate`

The Users:
1.  **Name =** 'testUser1', **Email =** 'user1@gmail.com', **Password =**'password123',
2.  **Name =**'testUser2', **Email =**'David@gmail.com', **Password =**'Password456',
3.  **Name =**'testUser3', **Email =**'User3@gmail.com', **Password =**'User3Password'

Resolves: #28

### CODE REVIEW

**Review by oriAdler — APPROVED** [review]
LGTM

**Review by VeraTs — APPROVED** [review]
LGTM

**Review by ellaml — APPROVED** [review]

File: homepage/migrations/0002_user_test_data.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    dependencies = [
+        ('homepage', '0001_initial'),
+    ]
+
+    def generate_user_test_data(apps, schema_editor):
+        from homepage.models import User
+
+        users_test_data = [
+            ('testUser1', 'user1@gmail.com', 'password123'),
+            ("testUser2", "David@gmail.com", "Password456"),
```
Comment (kobihk) [inline homepage/migrations/0002_user_test_data.py]: please try to keep your string consistent by surround all with one apostrophe(') or two apostrophes (")

in my opinion it looks better

File: homepage/migrations/0002_user_test_data.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            ("testUser3", "User3@gmail.com", "User3Password")
+        ]
+
+        with transaction.atomic():
+            for user_name, user_email, user_password in users_test_data:
+                User(name=user_name, password=user_password, email=user_email).save()
+
+    operations = [
+        migrations.RunPython(generate_user_test_data),
+    ]
```
Comment (kobihk) [inline homepage/migrations/0002_user_test_data.py:25]: please add one empty line in the end of the file

**Comment by LeonRabinovich** [comment]
@kobihk 

- I fixed the strings and changed all of them to (') 

- regarding the empty line at the end of the file - its there but it is not shown in github (its the same for all of the files which we previously commited and are in the repository)

Also the flake8 test passed about this.

**Pull request merged**